curriculum/challenges/english/01-responsive-web-design/css-grid-magazine/part-031.md

```diff
@@ -0,0 +1,265 @@
+---
+id: 6144f1410990ea17187a722b
+title: Part 31
+challengeType: 0
+dashedName: part-31
+---
+
+# --description--
+
+Now you can style the layout of your grid. CSS Grid is similar to Flexbox in that it has a special property for both the parent and child elements. 
+
+In this case, your parent element is the `main` element. Set the content to have a three column layout by adding a `grid-template-columns` property with a value of `1fr 94rem 1fr`. This will create three columns where the middle column is `94rem` wide, and the first and last columns are both 1 fraction of the remaining space in the grid container.
```

Comment:
```suggestion
In this case, your parent element is the `main` element. Set the content to have a three-column layout by adding a `grid-template-columns` property with a value of `1fr 94rem 1fr`. This will create three columns where the middle column is `94rem` wide, and the first and last columns are both 1 fraction of the remaining space in the grid container.
```